.github/workflows/reusable-semgrep-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: Semgrep
+
+on:
+  workflow_call:
+    inputs:
+      github_event_name:
```

Comment:
Can the context be inherited?

Reply:
We can actually inherit the context here, removing them